INSTALL.md

```diff
@@ -39,7 +39,31 @@ $ sudo make install
 </pre>
 
 note: The `sudo` is very important for the `make install`. Failure to do this
-will result in a non-functioning or semi-functioning installation.
+will result in a non-functioning or semi-functioning installation. This is required
+due to Singularity requiring the `SUID` bit to be set, even if the user installs
+into a directory in which she has full priviledges.
+
+
+### Using nixpkgs to build
```

Comment:
Follow up from Slack chat...

* Rebase against `development-2.x`
* Remove the `Using nixpkgs to build` section

Hrrmm...  As for linking examples.... I suppose that if we crate a new file `PKG_EXAMPLES.md`  (or similar), and within that have some known examples listed like:

```
## Packaging
For example Singularity packaging see:
Example [RPM Spec File](https://github.com/singularityware/singularity/blob/master/singularity.spec.in)
Example [Debian Package Structure](https://github.com/singularityware/singularity/blob/master/debian)
Example [Alpine examples](https://github.com/singularityware/singularity/blob/master/alpinelinux)

### Tools
[easybuild](https://github.com/easybuilders/easybuild-easyconfigs/tree/master/easybuild/easyconfigs/s/Singularity)
[nixpkgs](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/virtualization/singularity/default.nix)
```

That would probably be fine. But we were mostly trying to keep the INSTALL.md minimal.